redaxo/src/core/tests/sql/sql_select_test.php

```diff
@@ -63,6 +63,48 @@ public function testGetRow()
         }
     }
 
+    public function testQueryType()
```

Comment:
Ich glaube, ich würde es eher in der allgemeinen `sql_test.php` platzieren, nicht hier in der speziellen Testklasse für SQL-Klasse für SELECT-Abfragen.

Reply:
ich verschiebe es dann